packages/common/src/time-base.ts

```diff
@@ -0,0 +1,6 @@
+import { StringValue } from 'ms';
+
+/**
+ * A duration, expressed either as a number of milliseconds, or as a {@link https://www.npmjs.com/package/ms | ms-formatted string}.
+ */
+export type Duration = StringValue | number;
```

Comment:
I don't mind the refactor, but why?

Reply:
Circular import

Reply:
I see. Can you try instead to move the `WorkflowExecutionAlreadyStartedError` class out of `errors.ts` to `failure.ts`? I think that should break the cycle in a more natural way, given that `WorkflowExecutionAlreadyStartedError` is indeed a `TemporalFailure`, not a generic Error.